.prettierrc.json

```diff
@@ -0,0 +1,7 @@
+{
+  "printWidth": 96,
+  "semi": false,
+  "singleQuote": true,
+  "arrowParens": "always",
+  "trailingComma": "all"
```

Comment:
- 80 is too narrow for react.
- most of the case, semicolon is useless and verbose
- single quotes makes your finger's life easier
- number of the parameters in arrow function often change
- number of the entries in the object often changes, not appending comma will causes verbose diff